docs/api/minari_functions.md

```diff
@@ -10,7 +10,7 @@ minari_dataset/episode_data
 ## Create Minari Dataset
 
 ```{eval-rst}
-.. autofunction:: minari.create_dataset_from_collector_env
+.. autofunction:: env.create_dataset
```

Comment:
I don't think this generates the documentation properly
Can you check? You can build the doc locally

Reply:
Couldn't spot it since I had failures in building the docs. Now removed it since it is no longer required due to the deprecation.